config/routes.rb

```diff
@@ -17,4 +17,5 @@
 
   get "/:id/:node_name", to: "session_answers#show", as: :session_flow
   get "/:id/:node_name/next", to: "session_answers#update", as: :update_session_flow
+  delete "/:id", to: "session_answers#destroy", as: :delete_session_flow
```

Comment:
This doesn't quite make sense as a URL. Sending a delete action to /<flow_name>, you'd expect it to delete the flow ahah. If we are changing to a link, I think also it needs to be a GET request. So maybe something like: 
```suggestion
  get "/:id/clear-session", to: "session_answers#clear_session", as: :clear_session
```

Reply:
Hi @theseanything - resolved - thanks 👍